sdk/eventhubs/azure_messaging_eventhubs/src/common/recoverable/connection.rs

```diff
@@ -23,6 +24,8 @@ use azure_core_amqp::{
     AmqpManagement, AmqpReceiver, AmqpReceiverApis, AmqpReceiverOptions, AmqpSender,
     AmqpSenderApis, AmqpSession, AmqpSessionApis, AmqpSessionOptions, AmqpSource, AmqpSymbol,
 };
+#[cfg(feature = "test")]
```

Comment:
Why `feature = "test"` and not just `test`?

Reply:
Because [test] does not work across package boundaries. I need a feature to enable test hooks that are used across package boundaries (or from tests in the `test` directory.